### ISSUE TITLE
[bug] cargo tauri info always shows "Webview2 & Visual Studio Build Tools" not installed

### ISSUE THREAD

**Issue by xuchaoqian** [tauri-apps/tauri#3113]
### Describe the bug

cargo tauri info always shows "Webview2 & Visual Studio Build Tools" not installed. 

But actually I have installed them already:
![image](https://user-images.githubusercontent.com/1064052/147388294-a21aed88-9cbf-472a-b19f-c65c206bc4a2.png)

![image](https://user-images.githubusercontent.com/1064052/147388314-4747d8d8-95d6-4323-99f6-8ebd8c2414bd.png)

### Platform and versions

```shell
Operating System - Windows, version 10.0.19043 X64
Webview2 - Not installed
Visual Studio Build Tools - Not installed

Node.js environment
  Node.js - 14.18.2
  @tauri-apps/cli - 1.0.0-beta.10
  @tauri-apps/api - 1.0.0-beta.8

Global packages
  npm - 6.14.15
  pnpm - Not installed
  yarn - 1.22.17

Rust environment
  rustc - 1.57.0
  cargo - 1.57.0

Rust environment
  rustup - 1.24.3
  rustc - 1.57.0
  cargo - 1.57.0
  toolchain - stable-x86_64-pc-windows-msvc (environment override by RUSTUP_TOOLCHAIN)

App directory structure
/build
/node_modules
/public
/src
/src-tauri

App
  tauri.rs - 1.0.0-beta.8
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
  distDir - ../build
  devPath - http://localhost:3000/
  framework - React
```

### Additional context
1) I use the next tauri-cli: cargo install tauri-cli --git https://github.com/tauri-apps/tauri.git --branch next
2) And the installer will download Webview2 runtime every time, even if I have run the installer many times:
![image](https://user-images.githubusercontent.com/1064052/147388525-dbdde012-ad02-47cb-a843-3f652e1c8082.png)

**Comment by amrbashir** [comment]
Microsoft has finally released a non-elevated version of Webview2 installer and I suspect they changed the registry location. I will confirm later.

**Comment by FabianLars** [comment]
@amrbashir #3076
That's only for the WIX installer and probably not correct, but i linked relevant issues in the PR description.

Also unrelated to the build tools i guess

**Comment by amrbashir** [comment]
Yeah I had the build tools problem before and I plan to fix it when I get back.

As for webview2 issue, both the cli and bundler use the same way to detect Webview2 installation so the fix should work for both.

**Comment by FabianLars** [comment]
re-opening because i only fixed the webview2 detection, but wasn't able to stop github from closing this after i added the reference...

**Comment by xuchaoqian** [comment]
Thank you @FabianLars @amrbashir 

Feedback: 
1) Now the installer only downloads Webview2 runtime for the first time.
2) Now cargo tauri info shows: 
```
Operating System - Windows, version 10.0.19043 X64
Webview2 - 96.0.1054.62
Visual Studio Build Tools - Not installed
WARNING: no lock files found, defaulting to npm

Node.js environment
  Node.js - 14.18.2
  @tauri-apps/cli - 1.0.0
```-beta.10
  @tauri-apps/api - Not installed

Global packages
  npm - 6.14.15
  pnpm - Not installed
  yarn - 1.22.17

Rust environment
  rustc - 1.57.0
  cargo - 1.57.0

Rust environment
  rustup - 1.24.3
  rustc - 1.57.0
  cargo - 1.57.0
  toolchain - stable-x86_64-pc-windows-msvc (environment override by RUSTUP_TOOLCHAIN)

App directory structure
/.git
/bin
/public
/src
/webpack

App
  tauri.rs - 1.0.0-beta.8
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
  distDir - ../build
  framework - React
  bundler - Webpack

**Comment by PrimeDominus** [comment]
I've been able to get Tauri working on Windows 10 using the standard instructions explained in the docs on https://tauri.studio however when trying again using `cargo install tauri-cli --git https://github.com/tauri-apps/tauri.git --branch next` I get the following when running `cargo tauri info`

```
Operating System - Windows, version 10.0.19042 X64
Webview2 - 97.0.1072.69
Visual Studio Build Tools - Not installed

Node.js environment
  Node.js - 16.13.1
  @tauri-apps/cli - 1.0.0-beta.10
  @tauri-apps/api - 1.0.0-beta.8

Global packages
  npm - 8.1.2
  pnpm - Not installed
  yarn - 1.22.5

Rust environment
  rustc - 1.60.0-nightly
  cargo - 1.60.0-nightly

Rust environment
  rustup - 1.24.3
  rustc - 1.60.0-nightly
  cargo - 1.60.0-nightly
  toolchain - nightly-x86_64-pc-windows-msvc (environment override by RUSTUP_TOOLCHAIN)

App directory structure
/.git
/.vscode
/node_modules
/public
/src
/src-tauri

App
  tauri.rs - 1.0.0-beta.8 (no lockfile)
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
  distDir - ../build
  devPath - http://localhost:3000/
  framework - React
```

**Comment by amrbashir** [comment]
it will be fixed in the next release once #3144 is merged
